src/test/java/seedu/duke/logic/parser/ParserTest.java

```diff
@@ -107,14 +117,37 @@ void extractType_dineCommand_returnsTrue() throws CustomException {
         assertTrue(isOngoing);
     }
 
+    @Test
+    void extractType_listFavCommand_returnsFalse() throws CustomException {
+        FavList list = new FavList();
+        String fav = "/help";
+        Parser p = new Parser(fav);
+        p.extractType();
+        p.setUserInput("/addfav");
+        p.extractType();
+        p.setUserInput("/listfav");
```

Comment:
Is there supposed to be an assertEqual here?

Reply:
Ah yes, I resolved the conflict and forgot to add in the assert.